doc/source/contribution_and_api/unit_testing.rst

```diff
@@ -0,0 +1,171 @@
+.. _ref_unit_testing_contributing:
+
+Unit Testing
+============
+
+Unit tests are functions that validate the software by testing that the
+logic implemented inside a certain method, class or module is
+working as expected. They should try to be as atomic and 
+independent as possible.
```

Comment:
```suggestion
Unit tests validate the software by testing that the logic
implemented inside a certain method, class, or module is
working as expected. They should be as atomic and 
independent as possible.
```
Wondering if you meant anatomical rather than atomic?